tests/chainer_tests/utils_tests/test_sparse.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+
+import numpy
+
+from chainer import testing
+from chainer import utils
+
+
+def _setup_tensor(_min, _max, shape, dtype, threshold=None):
+    y = numpy.random.uniform(_min, _max, shape).astype(dtype)
+    if threshold is not None:
+        y[numpy.where(y < threshold)] = 0
```

Comment:
I think you can simplify it to `y[y < threshold] = 0`.

Reply:
Thank you! That's better.